neo/Plugins/IConsensusMetricsPlugin.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Neo.Network.P2P.Payloads;
+
+namespace Neo.Plugins
+{
+    public interface IConsensusMetricsPlugin
+    {
+        void AnalyzePrepareRequest(ConsensusPayload payload);
```

Comment:
i think that is better without `Analyze` word
 Just `OnPrepareXX`

Reply:
Better, let's remove it.